fluentui_tablayout/src/main/java/com/microsoft/fluentui/tokenized/navigation/TabBar.kt

```diff
@@ -76,6 +78,7 @@ fun TabBar(
                     onClick = tabData.onClick,
                     accessory = tabData.badge,
                     showIndicator = showIndicator,
+                    indicatorStyle = indicatorStyle,
```

Comment:
since we are already passing this separately, why have we included this in tabItemTokens?

Reply:
Will remove arguments for indicatorStyle and Text alignment in the upcoming PR and merge them into tabItemInfo to pass as a single argument instead